README.md

```diff
@@ -56,10 +62,10 @@ Go to [Get Started](./docs/user/get-started.md) page to try out Vald :)
     - Vald can be easily installed in a few steps.
 
 - Highly customizable
-    - You can configure the number of vector dimension, the number of replica and etc.
+    - You can configure the number of vector dimensions, the number of replica and etc.
 
 - Multi language supported
-    - Golang, Java, Nodejs and python is supported.
+    - Golang, Java, Nodejs, and python are supported.
```

Comment:
![badge](https://img.shields.io/badge/review-suggestion-blue.svg?style=flat-square)
```suggestion
    - Golang, Java, Clojure, Node.js, and Python clients are supported.
    - gRPC APIs can be called by any other languages that supports gRPC.
    - REST APIs are provided.
```

Reply:
maybe this is better
```suggestion
    - Golang, Java, Clojure, Node.js, and Python client library is supported.
    - gRPC APIs can be triggered by any programming languages which support gRPC.
    - REST API is also supported.```